source/agora/test/Fee.d

```diff
@@ -214,10 +214,6 @@ unittest
     {
         createAndExpectNewBlock(Height(block_idx));
     }
-
```

Comment:
point 5
https://github.com/bpfkorea/agora/issues/246#issuecomment-791075528

Reply:
Does this mean a node can be fed blocks with wrong CB txs? Do we just trust the peer when we are catching up?

Reply:
> Does this mean a node can be fed blocks with wrong CB txs? Do we just trust the peer when we are catching up?

We are still checking if at least 50% of the validators signed the block, so not just trusting only 1 peer. 

------ background

However, Mathias described an attack before
1. in 2020, lets say we have 20 validators, 2 are malicious, 8 are lazy(they don't sign blocks), and 10 are well-behaving. Those 10 well-behaving is able to create a block in 2020.
2. in 2022, 8 validators(from 2020) sell their private keys(6 previously lazy ones and 2 previously well-behaving one are the sellers).
3. now the person who bought those private keys can dupe a 'catching up' a validator.

Our block reward(based on validators signing blocks) will try to minimize the number of lazy validators, so let's in 2020 we didn't have 8 lazy validators, just 2, so the block was signed by 2+10=12 validators... With block reward in mind the person in 2022 have to buy more private keys, because we will also change our catchup logic at some point to the following one:

each validator needs to accept the block with the MOST signatures at a specific height

----- end of background

so if we implement all above, then a catching up validator can only be duped if someone can create a block with let's say 60%-70% of all signatures